messages/context.json

```diff
@@ -4,5 +4,7 @@
   "admin/editor.link.href.description": "The actual link e.g. \"https://www.google.com\", \"/about-us\", etc",
   "admin/editor.product-link.title": "Product Link",
   "admin/editor.link.target.title": "Where to open the link",
-  "admin/editor.link.target.description": "You can check the options available at: https://developer.mozilla.org/docs/Web/HTML/Element/a"
+  "admin/editor.link.target.description": "You can check the options available at: https://developer.mozilla.org/docs/Web/HTML/Element/a",
+  "admin/editor.link.rel.attribute.title": "\"rel\" attribute",
+  "admin/editor.link.rel.attribute.description": "You can check the options available at: https://www.w3schools.com/tags/att_a_rel.asp"
```

Comment:
This is kinda of a nitpick, but could you use the same link that is used in the README? https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types